### PR TITLE
Prepare v0.9.0 release

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

And fix a couple of tiny bugs while I'm on it :)

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [x] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [x] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/PROCESS.md [hunk 1]
```diff
@@ -65,14 +65,16 @@ Try to estimate on the planning as best as you can.
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not mention the issue number but describe the fixed issue instead.
 3. Most of the time pull requests are related to some issues.
-   In such cases, pull request **description** should mention the issue number like `Closes #{issue_number}.`.
-   Do not use URLs like `https://github.com/org/repo/issue/123` even if they are rendered the same on GitHub.
+   In such cases, pull request **description** should mention the issue number
+   like `Closes #{issue_number}.` or `Closes org/repo#{issue_number}.`.
+   Do not use URLs like `https://github.com/org/repo/issue/123` or paths like `org/repo/issue/123`
+   even if they are rendered the same on GitHub.
```
Comment (AlekSi) [inline .github/PROCESS.md:71]: @rumyantseva

**Review by Fashander — DISMISSED** [review]
LGTM

**Review by Fashander — APPROVED** [review]

File: .github/PROCESS.md (hunk 1 above)
Comment (rumyantseva) [reply .github/PROCESS.md:71]: 👌

File: internal/handlers/pg/msg_saslstart.go [hunk 2]
```diff
@@ -52,15 +53,23 @@ func (h *Handler) MsgSASLStart(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
...
 	}
 
-	conninfo.Get(ctx).SetAuth(string(parts[1]), string(parts[2]))
+	authzid, authcid, passwd := parts[0], parts[1], parts[2]
+
+	// Some drivers (Go) send empty authorization identity (authzid),
+	// while others (Java) set it to the same value as authentication identity (authcid)
+	// (see https://www.rfc-editor.org/rfc/rfc4616.html).
+	// Ignore authzid for now.
+	_ = authzid
```
Comment (noisersup) [inline internal/handlers/pg/msg_saslstart.go:70]: So we assume/we are sure that we are still able to do authorization on every driver with just authcid, right?

**Review by noisersup — APPROVED** [review]
LGTM!

**Review by rumyantseva — APPROVED** [review]
LGTM

File: internal/handlers/pg/msg_saslstart.go (hunk 2 above)
Comment (AlekSi) [reply internal/handlers/pg/msg_saslstart.go:70]: I don't think `authzid` is actually used by MongoDB for authorization. There is nothing in MongoDB docs that suggest that authentication and authorization identities could be different

Comment (noisersup) [reply internal/handlers/pg/msg_saslstart.go:70]: Yeah that wouldn't make sense

**Pull request merged**